README.md

```diff
@@ -9,19 +9,18 @@ To run tests (from root of repo):
 
 1. Create a branch off `master` using the following convention:
 ```shell
-git checkout -b 1.6.0-release
+git checkout -b 1.7.0-release
 ```
 2. On a new branch based on the release branch, update CHANGELOG.md with any changes in this release and metadata.json with the new version number.
 3. Commit these changes and put up a PR against the release branch you created in Step 1 and get review.
 4. Once the changes have been approved and merged to the release branch, pull down the updated release branch and tag the module.
 ```shell
-git tag -a 1.6.0 -m "1.6.0"
+git tag -a 1.7.0 -m "1.7.0"
```

Comment:
It's fine you updated these, but they're supposed to be examples, so I don't think we need to worry about keeping them updated.

Reply:
Yea, it was just one of those "since I'm in here" kinda things.